package.json

```
@@ -60,10 +63,8 @@
     ]
   },
   "generator": {
-    "generator": ">=0.50.0 <2.0.0",
-    "filters": [
-      "@asyncapi/generator-filters"
-    ],
+    "renderer": "react",
+    "generator": ">=1.0.1 <2.0.0",
```

This should become:

```suggestion
    "generator": ">=1.1.0 <2.0.0",
```

Maybe not now but once we release Generator with React support. Otherwise, people will try to use it with 1.0.1 and it will not work.

---

Yes I know that. With 1.1.0 local setup doesn't work. I will apply this after release generator :)

---

You don't have to. Use `npm link`: https://docs.npmjs.com/cli/v6/commands/npm-link.

---

I mean that if I change to `1.1.0` version then local version throw error that I used template not compatible with generator, because generator is in `1.0.2` and template needs `1.1.0`. I will change it, don't worry :)

---

LOL, left the comment in the wrong place. Sorry.